### DISCUSSION TITLE
Unable to Generate Reports for dfkchain

### DISCUSSION THREAD

**Discussion by Lodewi** [pwillworth/dfkreport#25] [Q&A] [answered]
I'm unable to generate any reports for dfkchain due to RPC errors. Is there a workaround? Generating reports for Harmony works fine with the same address.

![dfkerror](https://user-images.githubusercontent.com/121200151/209006321-1df87b88-fbef-4e91-abc9-a8a80150aaae.png)

**Comment by pwillworth** [comment]
Looks like it's hitting some rate limits on the provider used to lookup transactions for DFK chain, tweaking it now

**Comment by pwillworth** [comment, marked as answer]
@Lodewi Ok, I think I got it dialed in now if you want to try again
